### PR TITLE
bpo-37531: Fix regrtest timeout for subprocesses

### PR DESCRIPTION
**Pull request by vstinner**
Co-Authored-By: Joannah nanjekye <joannah.nanjekye@ibm.com>

https://bugs.python.org/issue37531

### CODE REVIEW

**Comment by vstinner** [comment]
I couldn't push to PR #14679, so I forked the PR to add my change on top of it.

**Comment by tirkarthi** [comment]
Guess issue number should be 37531 in the PR title.

File: Lib/test/libregrtest/runtest.py [hunk 1]
```diff
@@ -179,6 +184,7 @@ def runtest(ns, test_name):
         FAILED           test failed
         PASSED           test passed
         EMPTY_TEST_SUITE test ran no subtests.
+        TIMEOUT          test timd out.
```
Comment (tirkarthi) [inline Lib/test/libregrtest/runtest.py]:
```suggestion
        TIMEOUT          test timed out.
```

**Comment by vstinner** [comment]
> Guess issue number should be 37531 in the PR title.

Oops, right! I also fixed the typo that you spotted.

**Pull request merged**